src/app/shell/personal-cabinet/user-config/user-config.component.ts

```diff
@@ -16,21 +14,16 @@ export class UserConfigComponent implements OnInit {
 
   @Select(RegistrationState.user)
   user$: Observable<User>;
-  authServer: string;
+  authServer: string = environment.stsServer;
+  link: string;
 
-  constructor(private dialog: MatDialog) { }
+  constructor() { }
 
   ngOnInit(): void {
-    this.authServer = environment.stsServer;
-  }
-
-  openDialogWindow(link: string): void {
-    const dialogRef = this.dialog.open(EditModalComponent, { data: link }
-    );
 
-    dialogRef.afterClosed().subscribe(result => {
-      console.log(`Dialog result: ${result}`);
-    });
+  }
 
+  onRedirect(link: string): void {
+    window.open(this.authServer + link, "windowName", "height=500,width=500");
```

Comment:
1. we use single quotes 
2. windowName is just for example, please use meaningful name